gradle testJavaScript failed (require nodejs > 15)

````
$ gradle testJavaScript
Configuration on demand is an incubating feature.

> Task :npmInstall
npm WARN tokenscript-attestation@0.0.1 No repository field.
npm WARN tokenscript-attestation@0.0.1 No license field.

audited 6 packages in 0.391s
found 0 vulnerabilities


> Task :testJavaScript FAILED
internal/process/esm_loader.js:74
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/a/IdeaProjects/attestation/src/main/javascript/SignedDevonTicket.js' imported from /home/a/IdeaProjects/attestation/src/test/javascript/ParserSerializerTest.js
    at finalizeResolution (internal/modules/esm/resolve.js:276:11)
    at moduleResolve (internal/modules/esm/resolve.js:699:10)
    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:810:11)
    at Loader.resolve (internal/modules/esm/loader.js:86:40)
    at Loader.getModuleJob (internal/modules/esm/loader.js:230:28)
    at ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:56:40)
    at link (internal/modules/esm/module_job.js:55:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':testJavaScript'.
> Process 'command '/home/a/IdeaProjects/attestation/.gradle/nodejs/node-v14.15.3-linux-x64/bin/node'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 7s
6 actionable tasks: 3 executed, 3 up-to-date
````

--- a/src/test/javascript/ParserSerializerTest.js
+++ b/src/test/javascript/ParserSerializerTest.js
@@ -2,7 +2,7 @@
 import { readFileSync } from 'fs';
 
 // testing SignedDevconTicket
-import { SignedDevconTicket } from "../../main/javascript/SignedDevonTicket.js"
+import { SignedDevconTicket } from "../../main/javascript/SignedDevconTicket.js"
 const der = readFileSync('build/test-results/signed-devcon-ticket.der')
 
 /* who can tell me why not just do this?

